react-components/components.md

```diff
@@ -0,0 +1,396 @@
+# Components
+
+## Introduction
+
+Sofia is a teacher, and she's creating a digital attendance webapp to help her at work. She wants to list all the students in her class.
+
+She knows that React will help her create the UI of her attendance webapp. But how does she turn her idea into React components?
+
+This lesson will follow the story of Sofia, who will learn the syntax to create and use two components.
+
+## Our Current Definition of Component
+
+**Components** in React are units of UI.
+
+All components can be rendered to the page, hold data, manage their own state, handle events, and contain other elements and components.
+
+Our understanding and definition of what a React component is and can do will grow over time. For now, let's learn the following.
+
+All components are:
+
+- Functions
+- Responsible for returning one JSX object, which determines how it's rendered
+
+## Using Components
+
+In order to "use" a component, there are three steps:
+
+1. Plan the component
+1. Define the component
+1. Render that component at least once
+
+## Planning the Component
+
+Sofia wants her attendance webapp to have the following UI displayed on one single page:
+
+- Information about the class
+- A list of students and their details (name, birthday)
+
+Both of these list items can create great React components! Sofia will plan to create two components.
+
+She'll start with creating the component to display a list of students and their details. While she's planning, Sofia will pick the name of her component.
```

Comment:
list item is confusing since of the bullets is a list itself.
make is more appropriate idiomatically than create here
using to as the discriminator (rather than that) leads to an ambiguous sentence parsing
```suggestion
Both of these features can make great React components! Sofia will plan to create two components.

She'll start with creating the component that displays a list of students and their details. While she's planning, Sofia will pick the name of her component.
```